tmail-backend/integration-tests/jmap/distributed-jmap-openpaas-integration-tests/src/test/resources/eml/calendar_reply_accepted-en.eml

```diff
@@ -0,0 +1,5 @@
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Subject: ACCEPTED: {{EVENT_TITLE}} @ {{EVENT_START_DATE}} ({{ATTENDEE}})
+
+{{ATTENDEE}} has accepted this invitation.
```

Comment:
Are those email templates needed? 
I bet it's not as when we uses OpenPaaS we actually do not send those emails.